src/Git_Updater/GU_Freemius.php

```diff
@@ -29,7 +29,7 @@ class GU_Freemius {
 	 * @return array|void
 	 */
 	public function init() {
-		if ( ! function_exists( 'gu_fs' ) ) {
+		if ( ! function_exists( __NAMESPACE__ . '\gu_fs' ) ) {
```

Comment:
I'm actually following how Freemius states to do this. The way the code is. I could probably omit the `function_exists()` check entirely as it is only loaded at the beginning and not called elsewhere.